data/test/macros/wml_unit_test_macros.cfg

```diff
@@ -161,6 +161,7 @@ bob#endarg
 # checks the number of strikes of all weapons a unit has
 # STRIKES is a comma-delimited list containing the number of strikes each weapon the unit has is expected to have
 # The implementation currently only supports one CHECK_STRIKES per unique ID1 per unit test
+# define NO_CHANCE_TO_HIT to skip setting the change to hit for both units to 100%
```

Comment:
This could be an optional argument instead of using a new mechanism.

Reply:
I don't believe that optional arguments can have multiline values.

Reply:
```
#define FOO
#arg BAR
[tag]
  value=default
[/tag]
#endarg
{BAR}
#enddef
{FOO}
{FOO (BAR=[tag]
  value=barvalue
[/tag])}
```
produces
```
[tag]
  value="default"
[/tag]
[tag]
  value="barvalue"
[/tag]
```

Reply:
I got an error when I tried it with the CHECK_STRIKES macro, but I'll try again.

Reply:
Fixed - I was putting the parenthesis around only the value rather than the whole thing. ie:
```
BAR=([tag]
  value=barvalue
[/tag])
```
vs
```
(BAR=[tag]
  value=barvalue
[/tag])
```

Reply:
Thanks for changing it, but this comment line needs to change too.

Reply:
"By default, both units chance to hit will be set to 100%. This can be overridden with the optional CHANCE_TO_HIT argument."